jdsol/1.0/_index.md

```diff
@@ -0,0 +1,53 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Debugging Support for Other Languages Spec 1.0"
+date: 2019-07-18T16:10:38-04:00
+summary: "First Version Spec for DSOL"
+---
+Jakarta Debugging Support for Other Languages provides a mechanism by which 
+programs executed under the JavaTM virtual machine but written in languages 
+other than the Java programming language, can be debugged with references to i
+the original source (for example, source file and line number references).
+
+* [Jakarta Debugging Support for Other Languages 1.0 Specification Document](./jdsol_1.0.pdf) (PDF)
+* [Jakarta Debugging Support for Other Language 1.0 Specification Document](./jdsol_1.0.html) (HTML)
+* [Jakarta  Debugging Support for Other Language 1.0 Javadoc](./apidocs)
+
+# Ballots
+
+## Release Review
+
+The Release Review Specification Committee Ballot concluded successfully on 2019-12-01 with the following results.
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                | &check; |         |          |
+|IBM                    | &check; |         |          |
+|Oracle                 | &check; |         |          |
+|Payara                 | &check; |         |          |
+|Red Hat                | &check; |         |          |
+|Tomitribe              | &check; |         |          |
+|EE4J PMC               | &check; |         |          |
+|Participant Members    |         | &check; |          |
+|Committer Members      | &check; |         |          |
+
+## Plan Review
+
+The Plan Review Specification Committee Ballot concluded successfully on 2019-09-31 with the following results.
+
+|                       |  Yes    | No  | Abstain  |
+|-----------------------|---------|-----|----------|
+|Fujitsu                | &check; |     |          |
+|IBM                    | &check; |     |          |
+|Oracle                 | &check; |     |          |
+|Payara                 | &check; |     |          |
+|Red Hat                | &check; |     |          |
+|Tomitribe              | &check; |     |          |
+|EE4J PMC               | &check; |     |          |
+|Participant Members    | &check; |     |          |
+|Committer Members      | &check; |     |          |
+
+# Compatible Implementations
+
+* Apache Marsupials 5.7
```

Comment:
Provide links to the actual binaries for these Compatible Implementations.

Reply:
cause Debugging has no java api, so there is no Compatible Implementations, will fix it as n/a